package.json

```
@@ -15,7 +15,7 @@
     "format:check": "npm run cmd:format -- --check",
     "jest": "NODE_ENV=test jest --env=jsdom",
     "jest:coverage": "npm run jest && open-cli ./coverage/index.html",
-    "jest:watch": "npm run jest --watch",
+    "jest:watch": "NODE_ENV=test jest --env=jsdom --watch",
```

nice! this is missing the `--` delimiter, so the appended arg is applied to `npm` instead of `jest`. another way to fix it would be:
```suggestion
    "jest:watch": "npm run jest -- --watch",
```

but yours is fine!

---

Great! I had just added the envs for consistency with the `jest` command, but we don't need them there, so we can keep it simple instead. 🙂